README.md

```diff
@@ -112,6 +113,35 @@ Default path to cache directory: `node_modules/.cache/uglifyjs-webpack-plugin`.
 
 Path to cache directory.
 
+### `cacheKeys`
+
+**webpack.config.js**
+```js
+[
+  new UglifyJsPlugin({
+    cache: true,
+    cacheKeys: (defaultCacheKeys, file, options) => {
```

Comment:
Remove the 3rd argument please

Reply:
/cc @ooflorent done, need approve :smile: